ftw/file/JSONDetails.py

```diff
@@ -0,0 +1,24 @@
+from Acquisition import aq_inner
+from ftw.file.content.file import File
+from Products.CMFCore.utils import getToolByName
+from Products.TinyMCE.adapters.JSONDetails import JSONDetails
+import simplejson as json
+
+
+class FtwJSONDetails(JSONDetails):
+
+    def getDetails(self):
+        details_json = super(FtwJSONDetails, self).getDetails()
+        details = json.loads(details_json)
+
+        utility = getToolByName(aq_inner(self.context), 'portal_tinymce')
+        image_portal_types = utility.imageobjects.splitlines()
+
+        # patch default scale on ftw.file
+        if isinstance(self.context, File):
```

Comment:
since this adapter is registered with ``for="ftw.file.interfaces.IFile"``, this condition should never be ``False``, right? I think it can be removed.